internal/inbox/schema.sql

```diff
@@ -5,7 +5,7 @@ CREATE TYPE content_type AS ENUM(
 
 CREATE TABLE IF NOT EXISTS inbox_message(
     id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
-    posted_by UUID NOT NULL references units(id),
+    posted_by UUID NOT NULL references units(id) ON DELETE CASCADE,
```

Comment:
It doesn’t seem reasonable to clear all messages from the unit in everyone’s inbox after the unit is deleted.